pages/release-notes.mdx

```diff
@@ -122,6 +122,27 @@ Memgraph (v2.11.1 and lower).
 </Callout>
 
 
+
+## Lab v2.15.0 - July 3, 2024
+
+{<h3> New features and improvements </h3>}
+
+- Share a query with other users with access to the same
+  Lab and Memgraph. Works only on Lab in Docker and with
+  enterprise license.
+- You can configure quick-connect to connect to Memgraph
+  with an encrypted connection with the new environment
+  variable `QUICK_CONNECT_MG_IS_ENCRYPTED`.
+- Size of the Lab Docker image has been decreased by 5x.
```

Comment:
Maybe replace with :

The size of the Lab Docker image has been reduced to one fifth of its original size.

Reply:
Yep, makes sense. I will change it.